### DISCUSSION TITLE
怎么给仪表盘页面左侧栏增加按钮？

### DISCUSSION THREAD

**Discussion by Tayeusym** [bs-community/blessing-skin-server#600] [Q&A] [answered]
我想给左侧栏添加按钮，但不知道怎么添加。现在的几个按钮是自带的。看了用户手册还是不知道怎么搞。

**Comment by tnqzh123** [comment, marked as answer]
https://bs-plugin.netlify.app/guide/bootstrap.html#%E6%B7%BB%E5%8A%A0%E8%8F%9C%E5%8D%95%E9%A1%B9

**Reply by Tayeusym** [reply]
您好，我那个按照教程写了，添加之后访问个人主页仪表盘后上方显示提示插件存在错误，无法加载。怎么办？我要菜单的按钮只写了bootstrap.php文件。请问怎么解决？

**Reply by tnqzh123** [reply]
![image](https://github.com/bs-community/blessing-skin-server/assets/16630630/56fdd44e-65ef-4f7e-a51e-b04eab9a973a)
